docs/user-docs/latest/media-library/organizing-assets-with-folders.md

```diff
@@ -18,16 +18,21 @@ Downgrading from Strapi v4.3.0-beta to v4.2.0 will delete folders of the Media L
 
 By default, the Media Library displays folders and assets created at the root level. Clicking a folder navigates to this folder, and displays the following elements:
 
-- the folder title and the number of subfolders and assets the current folder contains (1)
+- the folder title and a breadcrumb line to navigate through folders (1)
```

Comment:
@maevalienard Not sure how to call it 🤔 Breadcrumbs? breadcrumb line? something else?
Tried different namings in this PR so you can get an idea, but in the end of course we'll make it consistent :-)

Reply:
Thanks, fixed!